source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -407,6 +407,7 @@ Make the jogwheel ring blink when the track playing is near the end
 Use the mixer to control input when using :hwlabel:`SHIFT`                                     `MixerControlsMixAuxOnShift`                false                                                                                                             true/false                                                                            Make the :hwlabel:`GRID` button blinking when the playback goes over a detected beat
 Number of samples used for jogwheel speed                                                      `WheelSpeedSample`                          3                                                                                                                 1..50                                                                                 Number of samples used to determine the jogwheel movement. A higher value will increase precision but latency too, and vice-versa
 Replace the sampler tab by a beatloop roll tab                                                 `UseBeatloopRollInsteadOfSampler`           false                                                                                                             true/false                                                                            Replace the sample tab as well of the sample feature with 8 beatloop roll
+:hwlabel:`Filter` is first QuickEffect                                                         `FilterIsFirstQuickEffect`                  false                                                                                                             true/false                                                                            Assign the first QuickEffect to the :hwbtn:`FILTER` button instead of button :hwbtn:`1. This option is useful if you are used to Filter being the primary quickeffect and have it at the top of your QuickEfect list.
```

Comment:
```suggestion
:hwlabel:`Filter` is first QuickEffect                                                         `FilterIsFirstQuickEffect`                  false                                                                                                             true/false                                                                            Assign the first QuickEffect to the :hwbtn:`FILTER` button instead of button :hwlabel:`1. This option is useful if you are used to Filter being the primary quickeffect and have it at the top of your QuickEfect list.
```

Reply:
I think you are also missing a backtick after `1`

Reply:
Oh, I also missed that we should fix the term for Quick Effects:
QuickEffects :tm: 
or
Quick Effects
or
quick effects

(but not quickeffects)

Reply:
notes addressed manually